.travis.yml

```diff
@@ -24,4 +24,5 @@ env:
     - TEST_SET=base SKIP_INSTALL=true SKIP_STACK=true
     - TEST_SET=reconfig
     - TEST_SET=solo SKIP_STACK=true
+    - TEST_SET=counting SKIP_STACK=true
```

Comment:
This should not be its own top-level container -- they are too expensive to spin up for each little thing, it takes a good 10 minutes to just setup the build.  Likely should be combined with test_solo -- see testing/test_base for an example of what that might look like.

Reply:
Right, merged two tests into testing/test_topos.